Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
```

Comment:
Testing a reply.

---
In reply to: [53893790](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r53893790) [](ancestors = 53893790,53845000,53844945,53844845)